components/bases-locales/charte/partners.js

```diff
@@ -1,23 +1,28 @@
 import PropTypes from 'prop-types'
 import Partner from '@/components/bases-locales/charte/partner'
+import {orderBy} from 'lodash'
 
 import colors from '@/styles/colors'
 
-function Partners({partnersList, isAllPartners}) {
-  const companyPartners = partnersList.filter(partner => partner.isCompany === true)
-  const partners = isAllPartners ? partnersList.filter(partner => partner.isCompany === false) : partnersList
+const orderByName = partners => {
+  return orderBy(partners, [partner => partner.name], ['asc'])
+}
+
+function Partners({epci, companies, shuffledPartners}) {
+  const companiesPartners = companies && orderByName(companies)
```

Comment:
La fonction `orderByName` n'est pas vraiment nécessaire, `orderBy` peut être utilisé comme ceci
```suggestion
  const companiesPartners = companies && orderBy(companies, 'name', 'asc')
```